### PR TITLE
Release notes for TrueType functions and PyPy support

### PR DESCRIPTION
**Pull request by nulano**
Adds release notes for the following PRs released in 8.0.0 (#4764):

 * **Refactor font_getsize and font_render (#4910)**; "Other Changes" section; for issues #4553, #4569, #2293
 * **Implement anchor for TrueType fonts (#4930)**; "API Changes" section; for issues #4511
 * **Add `getlength` and `getbbox` functions for TrueType fonts (#4959)**; "API Additions" section; for issues #4651, #3977, #4483, #3921, #4789; brief mention in "Other Changes" section for issue #4553
 * **Add support for CBDT and COLR fonts (#4955)**; "API Additions" section; for issues #3346, #891
 * **Drop support for old PyPy versions (#4964)**; "Backwards Incompatible Changes" section

I also listed the related issues for easier search.

### CODE REVIEW

File: docs/releasenotes/8.0.0.rst [hunk 1]
```diff
@@ -9,6 +9,12 @@ Python 3.5
 
 Pillow has dropped support for Python 3.5, which reached end-of-life on 2020-09-13.
 
+PyPy 7.1.x
+^^^^^^^^^^
+
+Pillow has dropped support for PyPy3 7.1.1.
+PyPy3 7.2.0, released on 2019-10-14, is now the minimum compatible version.
```
Comment (nulano) [inline docs/releasenotes/8.0.0.rst:16]: I'm not too happy with this wording, but can't think of a way to improve this.

From https://github.com/python-pillow/Pillow/pull/4964#issuecomment-706988596:
> I suggest in general we aim to support the latest release of PyPy?
> 
> (Excluding any Python versions that are EOL by CPython, i.e. Python 2.7.)
> 
> And as Pillow 8.0.0 is a major release, it's okay to remove old compat code?

Comment (hugovk) [reply docs/releasenotes/8.0.0.rst:16]: I think it's fine, thanks!

**Review by hugovk — APPROVED** [review]

**Comment by nulano** [comment]
Does **MSYS2 fixes (#4890)** need release notes?

**Comment by hugovk** [comment]
I think we can skip that one.

**Pull request merged**